app/controllers/application_controller/compare.rb

```diff
@@ -1277,20 +1277,20 @@ def comp_add_header_compressed(view, h, i)
     txt = h[:name].truncate(16)
     html_text = ""
     if %w(Vm VmOrTemplate).include?(@sb[:compare_db])
-      img = ActionController::Base.helpers.image_path("svg/vendor-#{h[:vendor].downcase}.svg")
+      img = ActionController::Base.helpers.image_path(h.decorate.fileicon)
       html_text << "<a title=\"#{h[:name]}\" href=\"/#{controller_name}/show/#{h[:id]}\">
                       <img src=\"#{img}\" align=\"middle\" border=\"0\" width=\"20\" height=\"20\"/>
                     </a>"
     elsif @sb[:compare_db] == "Host"
-      img = ActionController::Base.helpers.image_path("svg/vendor-#{h[:vmm_vendor].downcase}.svg")
+      img = ActionController::Base.helpers.image_path(h.decorate.fileicon)
       html_text << "<a href=\"/host/show/#{h[:id]}\">
                       <img src=\"#{img}\" align=\"middle\" border=\"0\" width=\"20\" height=\"20\" />
                     </a>"
     else
-      img = ActionController::Base.helpers.image_path("100/#{@sb[:compare_db].underscore}.png")
+      icon = h.decorate.fonticon
       html_text <<
         "<a href=\"/ems_cluster/show/#{h[:id]}\">
-          <img src=\"#{img}\" align=\"middle\" border=\"0\" width=\"20\" height=\"20\"/>
+          <i class=\"#{icon}\" align=\"middle\" border=\"0\" width=\"20\" height=\"20\"/>
```

Comment:
@epwinchell I'm not sure if this looks good or not, the fonticon is blue as the link's text (see screenshot)